app/views/nominations/request_nomination_invite/success.html.erb

```diff
@@ -5,9 +5,10 @@
 
     <%= govuk_panel title_text: "Your school has been sent a link", text: nil, classes: "govuk-!-margin-bottom-7" %>
```

Comment:
We’ve removed this panel in favour of a plain h1 - see [page in the prototype](https://manage-training-for-ects-5c9ed99fc0a8.herokuapp.com/check-school-email)
```suggestion
  <h1 class="govuk-heading-xl">Check your school email</h1>
```